src/System.Security.Cryptography.Cng/tests/OpenTests.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class OpenTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void OpenFromHandle1()
+        {
+
+            byte[] reExported;
+            using (CngKey keyOriginal = CngKey.Import(TestData.Key1, CngKeyBlobFormat.GenericPublicBlob))
```

Comment:
Based on reading the code here I'm assuming that `TestData.Key1` is an ECDH-P256 key; but it'd be nice if that was included in the name.